adb/adb_commands.py

```diff
@@ -40,7 +40,7 @@
 
 try:
   # Imported locally to keep compatibility with previous code.
-  from sign_m2crypto import M2CryptoSigner
+  from adb.sign_m2crypto import M2CryptoSigner
```

Comment:
After this PR, I'm going to put these into a sign folder with an __init__.py that does this import magic once